src/analyses/goto_check.cpp

```diff
@@ -915,8 +903,8 @@ void goto_checkt::float_overflow_check(
     {
       // Can overflow
       const isinf_exprt new_inf(expr);
-      const isinf_exprt op0_inf(expr.op0());
-      const isinf_exprt op1_inf(expr.op1());
+      const isinf_exprt op0_inf(to_binary_expr(expr).op0());
```

Comment:
lhs

Reply:
That method probably should exist, but doesn't.